src/redux.ml

```diff
@@ -918,6 +918,10 @@ and context () =
       else
         assumes_with_more_pending_splits <- assumes_with_more_pending_splits + 1;
     
+    method assert_term t =
+      self#register_pending_splits_count;
+      ignore (self#assume_internal t)
+    

```

Comment:
Please copy all statistics-gathering and verbosity code from method assume (but with "Redux.assume" -> "Redux.assert_term")